index.html

```diff
@@ -184,39 +189,29 @@ <h2 class="h2-size">I'm always interested in hearing about new projects, so if y
               <input type="email" id="mail" name="user_email" placeholder="Email-address" required>
             </div>
             <div>
-              <textarea id="msg" name="user_message" placeholder="Enter text here" rows="6" maxlength="500" required></textarea>
+              <textarea id="msg" name="user_message" placeholder="Enter text here" rows="5" maxlength="500" required></textarea>
             </div>
             <div>
               <button type="submit" class="footer-btn">Get in touch</button>
             </div>
           </form>
         </div>  
-        <div class="icons-container">
+        <div class="footer-icons">
           <ul class="social-media-icons d-flex flex-center">
             <li>
-              <a href="https://twitter.com/BenMukebo">
-                <img src="./images/icons/twitter.png" alt="twitter">
-              </a>
+              <a href="https://twitter.com/BenMukebo"><img src="./images/icons/twitter.png" alt="twitter"></a>
             </li>
             <li>
-              <a href="https://www.linkedin.com/in/kasongo-mukebo-ben-591720205/">
-                <img src="./images/icons/Linkedin-icon.png" alt="linkedin">
-              </a>
+              <a href="https://www.linkedin.com/in/kasongo-mukebo-ben-591720205/"><img src="./images/icons/Linkedin-icon.png" alt="linkedin"></a>
             </li>
             <li>
-                <a href="#">
-                    <img src="./images/icons/medium.png" alt="medium">
-                </a>
+              <a href="#"><img src="./images/icons/medium.png" alt="medium"></a>
```

Comment:
It would be nice if you increase a little bit the width of your cads in order to make them more similar as the Figma design you selected 😉 

- [ ] [OPTIONAL] Make the width of your cards more similar to the cards from the Figma design you selected.

Figma design | Yours

![image](https://user-images.githubusercontent.com/60164239/130194647-0bd734c9-5f55-4d10-aaf8-e2cc5026ef98.png)

Reply:
I used the same height from Figma `height: 474px;`